.gitignore

```diff
@@ -105,4 +105,7 @@ packages/dbaeumer*
 **/.DS_Store
 
 # Netlify
-.netlify
\ No newline at end of file
+.netlify
+
+# Vscode History Extension
+.history
```

Comment:
Is this the app https://marketplace.visualstudio.com/items?itemName=xyz.local-history

Would you recommend it? Should it be part of the .vscode/extensions.json?

Reply:
I'm still experimenting.  I loved WebStorm's built in local history.  It saved my butt quite a few times when I royally messed up git